src/main/java/uk/gov/hmcts/reform/pip/account/management/service/MediaLegalApplicationService.java

```diff
@@ -0,0 +1,144 @@
+package uk.gov.hmcts.reform.pip.account.management.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.hmcts.reform.pip.account.management.database.AzureBlobService;
+import uk.gov.hmcts.reform.pip.account.management.database.MediaLegalApplicationRepository;
+import uk.gov.hmcts.reform.pip.account.management.errorhandling.exceptions.NotFoundException;
+import uk.gov.hmcts.reform.pip.account.management.model.MediaAndLegalApplication;
+import uk.gov.hmcts.reform.pip.account.management.model.MediaLegalApplicationStatus;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+public class MediaLegalApplicationService {
+
+    private final MediaLegalApplicationRepository mediaLegalApplicationRepository;
+
+    private final AzureBlobService azureBlobService;
+
+    LocalDateTime now = LocalDateTime.now();
+
+    @Autowired
+    public MediaLegalApplicationService(MediaLegalApplicationRepository mediaLegalApplicationRepository,
+                                        AzureBlobService azureBlobService) {
+        this.mediaLegalApplicationRepository = mediaLegalApplicationRepository;
+        this.azureBlobService = azureBlobService;
+    }
+
+    /**
+     * Get a list of all the applications.
+     *
+     * @return A list of all applications
+     */
+    public List<MediaAndLegalApplication> getApplications() {
+        return mediaLegalApplicationRepository.findAll();
+    }
+
+    /**
+     * Get a list of applications by the status.
+     *
+     * @param status The MediaLegalApplicationStatus enum to retrieve applications by
+     * @return A list of all applications with the relevant status
+     */
+    public List<MediaAndLegalApplication> getApplicationsByStatus(MediaLegalApplicationStatus status) {
+        return mediaLegalApplicationRepository.findByStatus(status);
+    }
+
+    /**
+     * Get an application by the application id.
+     *
+     * @param id The id of the application
+     * @return The application if it exists
+     */
+    public MediaAndLegalApplication getApplicationById(UUID id) {
+        Optional<MediaAndLegalApplication> returnedApplication = mediaLegalApplicationRepository.findById(id);
+        if (returnedApplication.isEmpty()) {
```

Comment:
optional objects have an isPresent method which is reccomended over the isEmpty, but for what you need you can use the orElse(), if youre only returning the get or a exception, would be best suited